docs/framework/migration-guide/retargeting/htmltextwriter_does_not_render_br__element_correctly.md

```diff
@@ -0,0 +1,11 @@
+### HtmlTextWriter does not render `<br/>` element correctly
+
+|   |   |
+|---|---|
+|Details|Beginning in the .NET Framework 4.6, calling <xref:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.String)> and <xref:System.Web.UI.HtmlTextWriter.RenderEndTag> with a <code>&lt;BR /&gt;</code> element will correctly insert only one <code>&lt;BR /&gt;</code> (instead of two)|
+|Suggestion|If an app depended on the extra <code>&lt;BR /&gt;</code> tag, <xref:System.Web.UI.HtmlTextWriter.RenderBeginTag(System.String)> should be called a second time. Note that this behavior change only affects apps that are targeting the .NET Framework 4.6, so another option is to target a previous version of the .NET Framework in order to get the old behavior.|
```

Comment:
This needs to be changed to "only affects apps that target the .NET Framework 4.6 or later,"